addons/crm/models/crm_team.py

```diff
@@ -25,6 +25,19 @@ class Team(models.Model):
         compute='_compute_opportunities',
         string='Amount of quotations to invoice', readonly=True)
     dashboard_graph_model = fields.Selection(selection_add=[('pipeline', 'Pipeline')])
+    dashboard_graph_period_pipeline = fields.Selection([
```

Comment:
Reading the file, it seems it is a related on dashboard_graph_period . What is its use ?